# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: 0527d227c6b6
# Full Hash: 0527d227c6b6f3d6648704a01bfd1c090b971ea7
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 967e68ed5f56 -r 0527d227c6b6 dom/script/moz.build
--- a/dom/script/moz.build	Fri Feb 18 12:45:52 2022 +0000
+++ b/dom/script/moz.build	Fri Feb 18 12:45:52 2022 +0000
@@ -19,37 +19,29 @@
 
 EXPORTS.mozilla.dom += [
     "AutoEntryScript.h",
-    "LoadedScript.h",
     "ModuleLoader.h",
-    "ModuleLoaderBase.h",
-    "ModuleMapKey.h",
     "ScriptDecoding.h",
     "ScriptElement.h",
-    "ScriptKind.h",
     "ScriptLoadContext.h",
     "ScriptLoader.h",
-    "ScriptLoadRequest.h",
     "ScriptSettings.h",
+    "ScriptTrace.h",
 ]
 
 UNIFIED_SOURCES += [
     "AutoEntryScript.cpp",
-    "LoadedScript.cpp",
     "ModuleLoader.cpp",
-    "ModuleLoaderBase.cpp",
-    "ModuleLoadRequest.cpp",
-    "ModuleMapKey.cpp",
     "nsIScriptElement.cpp",
     "ScriptElement.cpp",
     "ScriptLoadContext.cpp",
     "ScriptLoader.cpp",
     "ScriptLoadHandler.cpp",
-    "ScriptLoadRequest.cpp",
     "ScriptSettings.cpp",
 ]
 
 LOCAL_INCLUDES += [
     "/dom/base",
+    "/js/loader",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")